docs/destroy-cluster.md

```diff
@@ -0,0 +1,63 @@
+# Destroying a Cluster
+
+When a cluster is no longer needed and needs to be torn down, Rook has a [Cleanup guide](https://rook.io/docs/rook/latest/Getting-Started/ceph-teardown/) with instructions to tear it down cleanly. While following that document is highly preferred, there is a command that will automate the cleanup.
+The `destroy-cluster` command is used for destroy Rook CRs (custom resources) and override any finalizers that may be preventing the cleanup.
+
+## !!! Warning !!!
+**This command is not reversible**, and it will destroy your Rook cluster completely with guaranteed data loss.
+Only use this command if you are 100% sure that the cluster and its data should be destroyed.
+
+## `destroy-cluster` command
+
+To destroy a cluster, run the command:
+```bash
+$ kubectl rook-ceph -n rook-ceph destroy-cluster
+Warning: Are you sure you want to destroy the cluster in namespace rook-ceph? If absolutely certain, enter: yes-really-destroy-cluster
+```
+
+Any other response will cause the command to abort.
+
+## Example of destroying a cluster
+
+```bash
+$ kubectl rook-ceph -n rook-ceph destroy-cluster
+Warning: Are you sure you want to destroy the cluster in namespace rook-ceph? If absolutely certain, enter: yes-really-destroy-cluster
+yes-really-destroy-cluster 
+Info: proceeding
+Info: Getting resources kind cephblockpoolradosnamespaces
+Warning: resource cephblockpoolradosnamespaces was not found on the cluster
```

Comment:
I believe you changed it so this is an Info message instead of a Warning, right? 
```suggestion
Info: resource cephblockpoolradosnamespaces was not found on the cluster
```